nominal/core/channel.py

```diff
@@ -50,6 +53,14 @@ class Channel(HasRid):
     description: str | None
     _clients: ClientsBunch = field(repr=False)
 
+    @abc.abstractmethod
+    def set_unit(self, unit_symbol: str | Unit | None) -> None:
+        """Set the unit for the data contained within the channel
+
+        Args:
+            unit_symbol: Symbol of the unit to set for the channel's data
+        """
+
     def to_pandas(self) -> pd.Series[Any]:
         """Retrieve the channel data as a pandas.Series.
 
```

Comment:
this method doesn't work for series archetypes, fyi

Reply:
That sounds okay for now, I expect to be able to actually impl it for archetypes soon